code/modules/security_levels/security_level_datums.dm

```diff
@@ -77,45 +94,64 @@
 	lowering_to_configuration_key = /datum/config_entry/string/alert_red_downto
 	elevating_to_configuration_key = /datum/config_entry/string/alert_red_upto
 	shuttle_call_time_mod = ALERT_COEFF_RED
-
-/**
- * DELTA
- *
- * Station destruction is imminent
- */
-/datum/security_level/delta
-	name = "delta"
-	announcement_color = "purple"
-	sound = 'sound/misc/delta_alert.ogg' // Air alarm to signify importance
-	number_level = SEC_LEVEL_DELTA
-	elevating_to_configuration_key = /datum/config_entry/string/alert_delta
-	shuttle_call_time_mod = ALERT_COEFF_DELTA
-
+	disable_night_mode = TRUE
+	pod_access = TRUE
+	emergency_doors = TRUE
 
 /**
  * GAMMA
  *
- * Station is not having a very good fun time
+ * Station is under severe threat, but not threat of immediate destruction
  */
 /datum/security_level/gamma
 	name = "gamma"
 	announcement_color = "orange"
-	sound = 'sound/misc/gamma_alert.ogg'// Air alarm to signify importance
+	sound = 'sound/misc/gamma_alert.ogg'
 	number_level = SEC_LEVEL_GAMMA
 	elevating_to_configuration_key = /datum/config_entry/string/alert_gamma
-	shuttle_call_time_mod = ALERT_COEFF_GAMMA
-
-
+	lowering_to_configuration_key = /datum/config_entry/string/alert_gamma
+	shuttle_call_time_mod = ALERT_COEFF_DELTA
+	disable_night_mode = TRUE
+	pod_access = TRUE
+	emergency_doors = TRUE
+	allow_cryo = FALSE
 
 /**
  * EPSILON
  *
- * Station destruction is here, and you probably caused it, you monster.
+ * [REDACTED]
```

Comment:
What is this change for lol

Reply:
The original comment is for delta, not epsilon

Reply:
can you describe what it is for then